### ISSUE TITLE
Cannot find unlabeled.csv

### ISSUE THREAD

**Issue by wodo96** [anhaidgroup/deepmatcher#84]
I'm trying to run it on Colab and everything work fine, until i need to run this:

`unlabeled = dm.data.process_unlabeled(
    path='data/Abt-Buy/unlabeled.csv',
    trained_model=model)`

Obviously it give me error because i don't have this file.
I thought it would create unlabeled.csv by itself, but it doesn't. I take all datasets from your ReadMe. Where i can find this unlabeled.csv file? I mean, i want to use this system with other datasets, so i would know how to take unlabeled.csv file.

Thanks.

**Comment by sidharthms** [comment]
https://github.com/anhaidgroup/deepmatcher/tree/master/examples/sample_data has samples of unlabeled data, is that what you're looking for?

**Comment by wodo96** [comment]
Yeah, but how can i "create" an unlabeled file for other datasets? Like "abt-buy" etc etc or also for custom datasets.
Also how can i properly use your system using tableA and tableB? I didn't find anything.

Ot: i really appreciate your job! It's amazing.

**Comment by sidharthms** [comment]
The merge_data function in this Colab shows how to join two tables to perform matching: https://colab.research.google.com/drive/14oacgxGheumvekKUq_VpLqOMzEFbIAVM#scrollTo=qj-AXGxmR_gL
